diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -302,7 +311,53 @@ impl<F, S, W, O, L, Of> ForUpdateDsl for SelectStatement<F, S, NoDistinctClause,
             self.limit,
             self.offset,
             self.group_by,
-            ForUpdateClause,
+            ForUpdateClause {
```

Comment:
Can we move this to `ForUpdateClause::new`?

Reply:
Yep